Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,181 @@
+require "utils/git"
+
+describe Git do
+  before(:all) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+    file = "lib/blah.rb"
+    repo = Pathname.new("repo")
+    FileUtils.mkpath("repo/lib")
+    system "#{git} init"
+    FileUtils.touch("repo/#{file}")
+    File.open(repo.join("file").to_s, "w") { |f| f.write("blah") }
```

Comment:
I'll have to explicitly close the file as well if I write open and write separately. Why do you think `(repo/"file").write "blah"` is a better way?